client/app/telecom/pack/slots/voipEcoFax/pack-voipEcoFax.html

```diff
@@ -1,25 +1,23 @@
-<div data-ng-controller="PackVoipEcoFaxCtrl as VoipEcoFax">
+<div data-ng-controller="PackVoipEcoFaxCtrl as $ctrl">
     <div class="oui-tile__body"
-         data-ng-if="loaders.services">
+         data-ng-if="$ctrl.loaders.services">
         <div class="oui-tile__definition text-center">
             <oui-spinner></oui-spinner>
         </div>
     </div>
     <div class="oui-tile__body"
-         data-ng-if="!loaders.services && VoipEcoFax.services.length == 0">
+         data-ng-if="!$ctrl.loaders.services && $ctrl.services.length == 0">
```

Comment:
I know you just have changed the controller name but : 
```suggestion
         data-ng-if="!$ctrl.loaders.services && $ctrl.services.length === 0">
```